package.json

```diff
@@ -20,7 +20,7 @@
     "test-jest": "jest --bail --logHeapUsage",
     "test-windows": "npm run lint && tap test/windows/**/*.test.ts -R=spec --timeout=300",
     "test-jest-windows": "jest --bail --config test/windows/jest.config.js --logHeapUsage",
-    "unit-test": "tap test/system/**/*.test.ts test/lib -R=spec --timeout=300",
+    "unit-test": "tap test/system/**/*.test.ts test/lib/**/*.test.ts -R=spec --timeout=300",
```

Comment:
This seems odd actually, unit tests are also running the "system" tests? Perhaps we should only be testing only things from `test/lib` like you added? Not sure why we had until now unit tests doing the system tests 😄

Reply:
You're totally right! Let me change that whiles we're at it :-)